frontend/src/components/Login.jsx

```diff
@@ -0,0 +1,177 @@
+// Package imports
+import React, { Component } from "react";
+import { connect } from "react-redux";
+
+// Local imports
+import { actions } from "../store";
+import FacebookLogin from "react-facebook-login";
+
+class _Login extends Component {
+  constructor() {
+    super();
+    this.facebookLogin = this.facebookLogin.bind(this);
+  }
+
+  facebookLogin(response) {
+    console.log("fb response", response);
+    const { fbLogin } = this.props;
+    const { first_name, last_name, accessToken, email, id, picture } = response;
+    const user = {
+      firstName: first_name,
+      lastName: last_name,
+      email,
+      facebookId: id,
+      imageUrl: picture.data.url
+    };
+
+    const auth = { token: accessToken, user };
+    fbLogin(auth);
+  }
+
+  render() {
+    const { facebookLogin } = this;
+
+    return (
+      <div>
+        <div id="businessLogin">
+          <h2>For Business:</h2>
+          <a href="/auth/linkedin">
+            <img src="./assets/images/linkedin-signin.png" />
+          </a>
+        </div>
+        <div id="creatorLogin">
+          <h2>For Creators:</h2>
+          <FacebookLogin
+            appId="507675923424391"
+            fields="first_name, last_name,email,picture"
+            // autoLoad={true}
+            callback={facebookLogin}
+            icon="fa-facebook"
+            size="medium"
+            scope="public_profile, user_posts, instagram_basic, instagram_manage_insights"
+            textButton="Sign In With Facebook"
+            redirectUri="/"
+          />
+        </div>
+      </div>
+    );
+  }
+}
+
+const mapDispatchToProps = (dispatch, { history }) => ({
+  fbLogin: auth =>
+    //dispatch facebook auth & user data
+    dispatch(actions.attemptToLogin(auth, history))
+});
+
+const Login = connect(null, mapDispatchToProps)(_Login);
+
+export default Login;
```

Comment:
can we make this similar to how we export the other components? like `export default connect(null, mapDispatchToProps)(Login)`? so that we don't need to use _ when we name our components.